requirements.txt

```diff
@@ -0,0 +1,4 @@
+flake8==3.7.9
```

Comment:
I don't think flake8 is required here, it's just for local tests. Could you please remove and check that CircleCI job still passes?